customer-wallet/src/containers/Transactions/TransactionInfo/index.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react'
+import { string, object, func } from 'prop-types'
+import moment from 'moment'
+// components
+import { Card, CardHeader, CardContent, Divider } from '@material-ui/core'
+import { List, ListItem, ListItemIcon, ListItemText } from '@material-ui/core'
+import { withStyles, Avatar } from '@material-ui/core'
+import CircularProgress from '@material-ui/core/CircularProgress'
+import Review from 'components/Review'
+// helpers
+import { formatCurrency } from 'helpers/currencyFormat'
+import { calculateTotalSpent } from 'helpers/transactions'
+import { get } from 'lodash'
+// icons
+import BlockchainIcon from '@material-ui/icons/Search'
+import ExplorerIcon from '@material-ui/icons/Explore'
+import PaymentIcon from '@material-ui/icons/CallMade'
+import NotChluIcon from '@material-ui/icons/ErrorOutline'
+import LoadingIcon from '@material-ui/icons/Sync'
+import UnconfirmedIcon from '@material-ui/icons/Gavel'
+import ValidIcon from '@material-ui/icons/Done'
+import InvalidIcon from '@material-ui/icons/ErrorOutline'
+
+const styles = {
+    card: {
+        margin: '30px'
+    }
+}
+
+function openInNewTab(url) {
+    if (url) {
+        const win = window.open(url, '_blank');
+        win.focus();
+    }
+}
+
+class TransactionInfo extends Component {
+
+    static propTypes = theme => ({
+        transaction: object,
+        address: string,
+        convertSatoshiToBits: func,
+        convertFromBitsToUsd: func
+    })
+
+    render () {
+        const {
+            classes,
+            transaction,
+            address,
+            convertSatoshiToBits,
+            convertFromBitsToUsd,
+            review,
+            editing
+        } = this.props
+
+        const amount = calculateTotalSpent(transaction, address)
+        const priceBits = convertSatoshiToBits(amount)
+        const confirmations = get(transaction, 'confirmations', 0)
+        const priceBitsFormatted = formatCurrency(priceBits)
+        const date = moment(transaction.received).calendar()
+        const isChlu = Boolean(review)
+        const reviewIsLoading = isChlu && review.loading
+        const reviewIsReady = isChlu && !review.loading
+        const reviewMultihash = isChlu ? review.multihash : null
+        const reviewIsUpdate = Boolean(review && review.last_version_multihash)
+        const reviewDate = reviewIsUpdate ? null : date
+        const chainExplorerLink = `${process.env.REACT_APP_BTC_EXPLORER_URL_TX}/${transaction.hash}`
+        const chluExplorerLink = isChlu ? `https://explorer.chlu.io/#/v/${reviewMultihash}` : null
+        
+        let status = {
+            primary: 'Confirmed',
+            icon: ValidIcon,
+            secondary: isChlu ? `Chlu data valid. ${confirmations} Blockchain confirmations` : 'Transaction confirmed'
```

Comment:
`Chlu data is valid. ${confirmations} transaction confirmations received.`

Small change to the text there.